torch/_export/__init__.py

```diff
@@ -683,6 +773,47 @@ def _export(
     constraints = constraints or []
     kwargs = kwargs or {}
 
+    if not strict:
+        assert isinstance(f, torch.nn.Module)
```

Comment:
why do we assert this if we just wrap f with another module?

Reply:
you're right, but I'd prefer to keep it here until we do need it.